<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CitySetting</class>
 <widget class="QDialog" name="CitySetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CitySetting</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>CitySetting</string>
   </property>
   <widget class="QComboBox" name="data_source">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Random</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FromFile</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> data source:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>city count:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>190</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
